test/unit/annotation_spec.js

```diff
@@ -107,6 +125,9 @@ describe("annotation", function () {
       idFactory: createIdFactory(/* pageIndex = */ 0),
       fontCache: new RefSetCache(),
       builtInCMapCache,
+      options: {
+        standardFontDataUrl: null,
+      },
```

Comment:
Given that `standardFontDataUrl: null,` is the default value, I don't think you should need this block now?